GrandCentralBoard/Widgets/Slack/SlackMessagesWidget.swift

```diff
@@ -25,10 +25,19 @@ final class SlackMessagesWidget: WidgetControlling {
     private let widgetView = SlackWidgetView.fromNib()
 
     let sources: [UpdatingSource]
-    var view: UIView { return widgetView }
+    let view: UIView
 
     init(source: SlackSource) {
         sources = [source]
+
+        let widgetTemplateViewModel = WidgetTemplateViewModel(title: "Slack".uppercaseString,
+                                                              subtitle: "",
+                                                              contentView: widgetView)
+        let layoutSettings = WidgetTemplateLayoutSettings(contentMargin: UIEdgeInsets(top: 26, left: 27, bottom: 27, right: 26))
```

Comment:
Taken from the design, but I think it can should be the same value (it seems design tools can lie ;)